samza-core/src/main/java/org/apache/samza/job/model/JobModelUtil.java

```diff
@@ -57,6 +67,42 @@
     return taskToSSPs;
   }
 
+
+  /**
+   * Converts the JobModel into a byte array into {@link MetadataStore}.
+   * @param jobModel the job model to store into {@link MetadataStore}.
+   * @param jobModelVersion the job model version.
+   * @param metadataStore the metadata store.
+   */
+  public static void writeJobModel(JobModel jobModel, String jobModelVersion, MetadataStore metadataStore) {
```

Comment:
This is currently ZK specific, right? 

Also, does this still need to be in a separate util class or can this be inlined / helper method in the caller?

Reply:
I think this is not specific to standalone. It will be applicable to YARN as well when it has support for versioning.

 I'm using these util functions in standalone integration test-cases to validate the `JobModel` published by a processors group.